src/bankAccount.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+function BankAccount(name, amount) {
+  this.name = name;
+  this.amount = amount;
+  this.history = [`Initial: ${amount}`];
+};
+
+BankAccount.prototype = {
+  getInfo() {
+    return `Name: ${this.name}, Amount: ${this.amount}$`;
+  },
+  addMoney(money, fromWhat) {
```

Comment:
некорректный второй аргумент, тебе нужно придумывать абстрактные названия
```suggestion
  addMoney(amount, purpose) {
```